docs/admin/integration/spark.md

```diff
@@ -0,0 +1,138 @@
+# Running SPARK jobs with Run:AI
+
+Spark has two modes for running jobs on kubernetes:
+
+* Using a CLI tool called `spark-submit` that submits raw pods.
+* CRD with operator.
+
+## CLI `Spark-submit`
+
+To run a Spark job on Kubernetes using the CLI:
+
+1. Download a pre-built spark with hadoop image from [here](https://spark.apache.org/downloads.html).
+2. Open the file, then go to its root to submit the jobs.
+
+### Cluster preparation
+
+Ensure that your Kubernetes cluster has a service account with permissions in the namespace that you want to run the jobs in. Use the following commands to launch the Spark demo:
+
+```
+k create ns spark-demo
+
+k create serviceaccount spark -n spark-demo
+
+k create clusterrolebinding spark-role --clusterrole edit --serviceaccount spark-demo:spark -n spark-demo
+```
+
+Change the namespace to `<your_K8s_Namespace>`.
+
+### Docker Images
+
+We need to build docker images and push them to either a public repository or load them to kind.
+
+To build the images run:
+
+```
+./bin/docker-image-tool.sh -t <image_name> build
+```
+
+An image named `<image_name` will be created.
+
+Then load it into your kind cluster:
+
+```
+kind load docker-image spark:<image_name> --name \<CLUSTER\_NAME\>
+```
```

Comment:
```suggestion
Then push the docker image to your repository:

```
docker tag spark:<image_tag> <MY_REPO>/spark:<image_tag>
docker push <MY_REPO>/spark:<image_tag>
```
```

Reply:
Did you want me to add the 
```
docker tag spark:<image_tag> <MY_REPO>/spark:<image_tag>
docker push <MY_REPO>/spark:<image_tag>
```
to the file? If so then where?